neofetch

```diff
@@ -2938,8 +2936,7 @@ get_distro_colors() {
             ascii_file="solaris"
         ;;
 
-        # TODO (Before PR is merged): Make this glob less greedy.
-        *"SUSE"*)
+        "openSUSE"* | "open SUSE")
```

Comment:
Shouldn't we add `"SUSE"*` too? To increase compatibility with SUSE Linux Enterprise?